### PR TITLE
feat(sggit): remove Version and introduce SHA and ShortSHA

### PR DESCRIPTION
**Pull request by viktorvoltaire**

### CODE REVIEW

**Review by alethenorio — APPROVED** [review]

**Comment by alethenorio** [comment]
@viktorvoltaire We should try and mark breaking changes in the commit message as per https://www.conventionalcommits.org/en/v1.0.0/

File: tools/sggit/tools.go [hunk 1]
```diff
@@ -24,15 +24,28 @@ func VerifyNoDiff(ctx context.Context) error {
 	return nil
 }
 
-func Version(ctx context.Context) string {
+func SHA(ctx context.Context) string {
```
[the remark this answers is not in the document]
Comment (viktorvoltaire) [reply tools/sggit/tools.go:27]: I kinda like that, WTDY @alethenorio

[the remark this answers is not in the document]
Comment (alethenorio) [reply tools/sggit/tools.go:27]: Both github and cloudbuild call it `sha` so I think  this is a good name to follow convention

[the remark this answers is not in the document]
Comment (thall) [reply tools/sggit/tools.go:27]: Fair enough

[the remark this answers is not in the document]
Comment (viktorvoltaire) [reply tools/sggit/tools.go:27]: True!

**Comment by viktorvoltaire** [comment]
> @viktorvoltaire We should try and mark breaking changes in the commit message as per https://www.conventionalcommits.org/en/v1.0.0/

Given that we are still on `v.0.` we reserve the rights for breaking changes. So we decided to leave out marking it until we hit `v1`

**Pull request merged**